frameworks/elastic/tests/test_shakedown.py

```diff
@@ -6,13 +8,15 @@
 import sdk_upgrade
 from tests.config import *
 
+log = logging.getLogger(__name__)
+
 FOLDERED_SERVICE_NAME = sdk_utils.get_foldered_name(PACKAGE_NAME)
```

Comment:
`sdk_utils` has been removed from `tests.config`, so one should probably import it here (failing TC test pointed this out)